web-components/src/components/icons/terassos/ComplianceBadge.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from '@mui/material';
+
+export default function ComplianceBadge() {
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      width="24px"
+      height="24px"
+      padding="3px"
+      borderRadius="50%"
+      fontSize="12px"
+      fontWeight={700}
+      sx={{
+        borderRadius: '40px',
+        border: '1px dashed var(--surface-stroke, #D2D5D9)',
+        background: 'var(--surface-selected-item-background, #EFEFEF)',
+        ml: 2,
+      }}
+      // eslint-disable-next-line lingui/no-unlocalized-strings
+    >
+      ha.
+    </Box>
```

Comment:
We should use Tailwind instead of MUI.

Reply:
I moved border radius and margin to use tailwind, but in this PR I don't yet have the Tailwind variables that I need to use here, so I can update it after we get the design tokens PR merged.